auth-agents-common/src/main/java/org/apache/atlas/policytransformer/CachePolicyTransformerImpl.java

```diff
@@ -174,6 +174,8 @@ public ServicePolicies getPoliciesDelta(String serviceName, Map<String, EntityAu
 
                 ArrayList<String> policyGuids = new ArrayList<>(policyChanges.keySet());
                 List<AtlasEntityHeader> allAtlasPolicies = getAtlasPolicies(serviceName, POLICY_BATCH_SIZE, policyGuids);
+                Date latestUpdateTime = allAtlasPolicies.stream().map(AtlasEntityHeader::getUpdateTime).max(Date::compareTo).orElse(null);
```

Comment:
This will be either first or last policy time.

Reply:
When it will be first and when it will be last?

Reply:
@nikhilbonte21 had suggested to use either last or first item from `allAtlasPolicies` rather than finding the maximum since it is already sorted. So I had noted it down. 

But the `allAtlasPolicies` array is sorted on `__timestamp` and not on updateTime, and it's an existing method, so edge elements won't give maximum updatTime.